.gitlab-ci.yml

```diff
@@ -178,3 +196,17 @@ westend:
   <<:                              *deploy-prod-refs
   <<:                              *kubernetes-env
   <<:                              *deploy-k8s
+
+# stage:                           publish-docker-description
+
+publish-bot-docker-image-description:
+  extends:                         .publish-docker-image-description
+  variables:
+    SHORT_DESCRIPTION:             "Faucet bot component"
+    COMPONENT:                     bot
+
+publish-server-docker-image-description:
+  extends:                         .publish-docker-image-description
+  variables:
+    SHORT_DESCRIPTION:             "Faucet server component"
+    COMPONENT:                     server
```

Comment:
@rzadp would this collide with #221?

Reply:
@Bullrich you're right changes introduced by @rzadp changes the game a bit :)
So I'll put this one on hold till #221 get merged. And then will adjust accordingly.
Good catch. Thank you.

Reply:
Updated to reflect changes from #221